bundle/Controller/UserController.php

```diff
@@ -149,6 +156,9 @@ public function editAction(
         Request $request
     ) {
         $user = $this->userService->loadUser($contentId);
+        if (!$this->permissionResolver->canUser('content', 'read', $user)) {
+            throw new CoreUnauthorizedException('content', 'read', array('userId' => $contentId));
```

Comment:
I think we should use `eZ\Publish\API\Repository\Exceptions\UnauthorizedException`